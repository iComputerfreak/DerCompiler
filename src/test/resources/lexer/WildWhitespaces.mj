               	           if
 (
         true
               	                 ) { System           .         out  	   .        println(
              			                      argument
                                        )                			          }
