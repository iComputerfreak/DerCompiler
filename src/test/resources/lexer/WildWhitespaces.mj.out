if
(
true
)
{
identifier System
.
identifier out
.
identifier println
(
identifier argument
)
}
EOF
